internal/osutils/autostart/autostart_lin.go

```diff
@@ -33,13 +30,35 @@ func (a *App) Enable() error {
 
 	dir, err := dirPath(autostartDir)
 	if err != nil {
-		return errs.Wrap(err, "Could not get autostart file")
+		return errs.Wrap(err, "Could not find autostart directory")
 	}
-	path := filepath.Join(dir, launchFile)
+	path := filepath.Join(dir, constants.TrayLaunchFileName)
 
-	if err = os.Symlink(appPath, path); err != nil {
-		return errs.Wrap(err, "Could not create symlink")
+	scut, err := shortcut.New(appinfo.TrayApp().Exec(), path)
+	if err != nil {
+		return errs.Wrap(err, "Could not construct autostart shortcut")
 	}
+
+	iconsDir, err := dirPath(constants.IconsDir)
```

Comment:
dirPath is not a great name, not obvious what it does

Reply:
There were two separate packages using this name, and both have been made more clear.